src/system/unwind.c

```diff
@@ -96,9 +96,18 @@ _Unwind_Ptr __gnu_Unwind_Find_exidx(_Unwind_Ptr pc, int* nrec) {
 	return (_Unwind_Ptr)&__exidx_start;
 }
 
+struct {
+	uint32_t[12] pcs;
```

Comment:
Wouldn't it make more sense for this to either be a pointer to an int array which is dynamically allocated, or as a linked list so that it can dynamically adjust the size based on the size of the trace?

Reply:
I used a fixed sized array because I was worried about running out of memory, which seems like something that would be bad in this scenario. I'm not sure how likely this is though, if it's highly unlikely then I can change this to use dynamic allocation.

Reply:
Also, is dynamic memory allocation still guaranteed to work after a data abort exception?

Reply:
After discussing on vtow I'm keeping this as a fixed sized array for now, I increased the array size to 20.